pandas/core/computation/expr.py

```diff
@@ -349,8 +350,8 @@ class BaseExprVisitor(ast.NodeVisitor):
     preparser : callable
     """
 
-    const_type: type[Term] = Constant
-    term_type = Term
+    const_type: ClassVar[type[Constant]] = Constant
```

Comment:
Should probably have one PR just focused on making ClassVars more explicit (type checkers have difficulties determining whether they are class or instance variables).

Reply:
Not sure I understand why you're using ClassVar here. I thought it was introduced in PEP 526 just to differentiate between class variables and instance variables within class definitions.